iocage_lib/ioc_stop.py

```diff
@@ -85,39 +85,7 @@ def __stop_jail__(self):
             silent=self.silent)
 
         if not self.force:
-            prestop, prestop_err = iocage_lib.ioc_common.runscript(
-                self.conf["exec_prestop"])
-
-            if prestop and prestop_err:
-                msg = f"  + Running prestop WARNING\n{prestop_err}"
-                iocage_lib.ioc_common.logit({
-                    "level": "WARNING",
-                    "message": msg
-                },
-                    _callback=self.callback,
-                    silent=self.silent)
-            elif prestop:
-                msg = "  + Running prestop OK"
-                iocage_lib.ioc_common.logit({
-                    "level": "INFO",
-                    "message": msg
-                },
-                    _callback=self.callback,
-                    silent=self.silent)
-            else:
-                if prestop_err:
-                    # They may just be exiting on 1, with no real message.
-                    msg = f"  + Running prestop FAILED\n{prestop_err}"
-                else:
-                    msg = f"  + Running prestop FAILED"
-
-                iocage_lib.ioc_common.logit({
-                    "level": "ERROR",
-                    "message": msg
-                },
-                    _callback=self.callback,
-                    silent=self.silent)
-
+            # TODO: Perhaps have jail(8) handle this ?
```

Comment:
Actually @skarekrow I think we definitely should now. As prestop would be handled by jail(8), it would not make sense to have `stop` executed before that

Reply:
As we discussed offline, it's prudent atm to handle all exec.* iocage side until we have some other breaking things for 1.2